src/plugins/intel_gpu/src/plugin/ops/split.cpp

```diff
@@ -14,53 +14,65 @@ namespace ov {
+            const auto outPartialShape = op->get_output_partial_shape(i);
+            if (outPartialShape.is_static()) {
+                NGRAPH_SUPPRESS_DEPRECATED_START
+                if (outPartialShape.size() != start_offset.size()) {
                     IE_THROW() << "Invalid dimesions in split layer: " << op->get_friendly_name()
                                     << " output: " <<  op->get_output_tensor_name(i);
                 }
-            }
-            NGRAPH_SUPPRESS_DEPRECATED_END
-
-            auto outTensor = tensor_from_dims(outPatialShape.to_shape(), 1);
-            auto offsetTensor = tensor_from_dims(startOffset, 0);
-            auto cropPrim = cldnn::crop(outLayerName, inputPrimitives, outTensor, offsetTensor, op_mode, i, num_splits);
+                for (size_t i = 0; i < input_pshape.size(); i++) {
+                    if ((outPartialShape[i].get_length() + static_cast<ov::Dimension::value_type>(start_offset[i])) > input_pshape[i].get_length()) {
+                        IE_THROW() << "Invalid dimesions in split layer: " << op->get_friendly_name()
+                                        << " output: " <<  op->get_output_tensor_name(i);
+                    }
+                }
+                NGRAPH_SUPPRESS_DEPRECATED_END
 
-            p.add_primitive(*op, cropPrim);
+                offset_tensors.push_back(tensor_from_dims(start_offset, 0));
 
-            for (size_t i = 0; i < inPartialShape.size(); i++) {
-                if (outPatialShape[i] != inPartialShape[i]) {
-                    startOffset[i] += outPatialShape.to_shape()[i];
+                for (size_t idx = 0; idx < input_pshape.size(); idx++) {
+                    if (outPartialShape[idx] != input_pshape[idx]) {
+                        start_offset[idx] += outPartialShape.to_shape()[idx];
+                    }
                 }
+            } else {
+                offset_tensors.push_back(cldnn::tensor(0));
             }
-        } else {
-            auto cropPrim = cldnn::crop(outLayerName, inputPrimitives, cldnn::tensor(1), cldnn::tensor(0), op_mode, i, num_splits);
+        }
 
+        return offset_tensors;
+    };
+
+    auto inputPrimitives = p.GetInputPrimitiveIDs(op);
+    auto offsetTensors = get_offset_tensors();
+    if (p.use_new_shape_infer() || op->is_dynamic()) {
+        cldnn::crop_ngraph_op_mode op_mode = cldnn::crop_ngraph_op_mode::variadic_split;
+        size_t num_splits = 1;
+        if (ngraph::is_type<ngraph::op::v1::Split>(op)) {
+            num_splits = ngraph::as_type_ptr<ngraph::op::v1::Split>(op)->get_num_splits();
+            op_mode = cldnn::crop_ngraph_op_mode::split;
+        }
+
+        for (size_t i = 0; i < op->get_output_size(); i++) {
+            const auto outPartialShape = op->get_output_partial_shape(i);
+            auto outTensor = outPartialShape.is_static()? tensor_from_dims(outPartialShape.to_shape(), 1) : cldnn::tensor(1);
```

Comment:
1) op->is_dynamic => output will be dynamic 
2) use_new_shape_infer => will eventually switched to use PartialShape only  

So there is no need for tensor_from_dims ?

Reply:
it is necessary for the case when p.use_new_shape_infer is true but op->is_dynamic() is false(If this case is not exist, I will remove this code). or do you mean that outTensor is changed from cldnn::tensor to PartialShape?

Reply:
I am curious why it is necessary. Did you encountered such a case? 
Also if the use_new_shape_infer is used, the default shape are expected to use PartialShape only, not the tensor_from_dims. Also it is expected to use ngraph shape infer in the calc_output_layout, instead of pre defined output shape.

Reply:
I understand what you mean. I fixed it.